platform_plugin_aspects/__init__.py

```diff
@@ -5,6 +5,6 @@
 import os
 from pathlib import Path
 
-__version__ = "0.10.0"
+__version__ = "0.10.1"
```

Comment:
I think that since this is a CI only change we can skip the version bump. Probably not worth it to do a release and upgrade in aspects for